src/toggle/Toggle.d.ts

```diff
@@ -0,0 +1,15 @@
+import { CSSProperties, PureComponent } from 'react'
+
+export type ToggleProps = {
+  value?: boolean,
+  onChange?: () => void, // TODO: not used
```

Comment:
actually it's used, `getValueLink` returns and object with a `requestChange` that can be the original `props.onChange` or `props.valueLink.requestChange`